universal-login-commons/src/core/utils/validateEmail.ts

```diff
@@ -0,0 +1,3 @@
+export const validateEmail = (email: string) => {
```

Comment:
add test

Reply:
Done

Reply:
if it validateEmail should throw error, else it should be 'isProperEmail'

Reply:
Done